tests/mocking.rs

```diff
@@ -132,6 +133,37 @@ fn manual() {
     assert_eq!(events, ActionEvents::COMPLETED);
 }
 
+#[test]
+fn external_mock() {
+    let mut app = App::new();
+    app.add_plugins((MinimalPlugins, EnhancedInputPlugin))
+        .add_input_context::<TestContext>()
+        .finish();
+
+    app.world_mut().spawn((
+        TestContext,
+        actions!(
+            TestContext[(
+                Action::<Test>::new(),
+                ExternallyMocked,
+                ActionMock::once(ActionState::Fired, true)
+            )]
+        ),
+    ));
+
+    app.update();
+
+    let mut actions = app
+        .world_mut()
+        .query::<(&Action<Test>, &ActionState, &ActionEvents)>();
+
+    // the action data should not be updated because of ExternallyMocked
+    let (&action, &state, &events) = actions.single(app.world()).unwrap();
+    assert!(!*action);
```

Comment:
```suggestion
    let (&action, &state, &events) = actions.single(app.world()).unwrap();
    assert!(!*action, "action shouldn't be updated because it marked as mocked externally");
```